Changes to README

ElectricFeet opened the pull request:
Moved sections; standardised syntax; clarified some areas for newbies

will-in-wi commented on README.md:
```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
+### SFTP
+
+If you're not happy navigating around your server's folders and files through SSH, you might find some of this process easier if you access your server with an FTP client over Secure FTP. WebFaction [also has this covered](https://docs.webfaction.com/user-guide/access.html#connecting-with-ftp).
+
```
I like this! The user will have to use SSH to set up the Cron job at some point, but if the bulk of the task is made easier by SFTP, that's great.

will-in-wi commented on README.md:
```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
 This utility works on [CentOS 6 and 7 boxes](https://docs.webfaction.com/user-guide/server.html#finding-your-server-s-operating-system). The CentOS 5 systems do not have a new enough OpenSSL to include the algorithms required. You may be able to make this work using rbenv and compiling openssl yourself. A tutorial for CentOS 5 is available here: https://github.com/will-in-wi/letsencrypt-webfactio ...
 
-All places where you need to substitute a value specific to your setup will be denoted with square brackets, e.g. `[mydomain.tld]`. There are cases where shell variables are used, such as `$HOME`. These should be typed verbatim.
+All places where you need to substitute a value specific to your setup will be denoted with square brackets, e.g. [yourdomain.com]. There are cases where shell variables are used, such as `$HOME`. These should be typed verbatim.
```
If you find this more comprehensible, I'll go with you on it. I guess my pedantic programmer brain has trouble making the second level of the domain (yourdomain) generic but not the top level domain (com). However, I'm guessing that if someone uses it for `.org` or something they'll understand what to change, while `.tld` is not understood.

ElectricFeet replied:
Yes, the purist pedant in me didn't like this one either; but my reasoning was exactly yours :-)

will-in-wi commented on README.md:
```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
+This will simplify the running of the LetsEncrypt WebFaction command, by setting some variables in advance.
 
-Then run the command `source $HOME/.bash_profile` to apply the new settings.
+After saving `~/.bash_profile`, run the command `source $HOME/.bash_profile` to apply the new settings.
```
Much clearer, thanks!

will-in-wi commented on README.md:
```diff
@@ -32,35 +32,40 @@ Cron is an application which will execute commands on a defined schedule. WebFac
+After saving `~/.bash_profile`, run the command `source $HOME/.bash_profile` to apply the new settings.
 
-Now, you can run `letsencrypt_webfaction` from the shell to get certificates. You can run this from any folder.
+Now, you are ready to run `letsencrypt_webfaction` from your SSH session to get certificates. See below for usage. 
```
Sheesh, you'd think I hadn't written decent prose in a while. This flows much better.

will-in-wi commented on README.md:
```diff
@@ -76,86 +81,109 @@ Once you have done so, install Ruby 2.1+ (probably 2.3.0 at time of writing). Th
+        In the case of a PHP site, such as Drupal or Wordpress, look for the folder with `index.php` in it. This is usually in `/home/[myuser]/webapps/[yourapp/]`.
 
-The certificate will be placed in `~/le_certs/example.com/[timestamp]/` and WebFaction will be emailed to install the certificate.
+        In the case of a Rails app, look for a folder called `public/`. If you are deploying your app with Capistrano, this could show up in `/home/myuser/webapps/[yourapp]/current/public/`. 
```
Markdown views indents as code examples. You can see in the [full rendered file](https://github.com/ElectricFeet/letsencrypt-webfaction/blob/ce384ffe7e291ae1637aa663e788c0681188db50/README.md) that you have to scroll left/right to read it all. I'd suggest using backticks (`) for the argument and then normal indentation for the description.

ElectricFeet commented:
I did the following:

Fixed all the formatting of the parameters. Seems to work OK.

Explained that the parameters listed are only the basic ones, and more detail is in the config file.

Moved (and tweaked) the `--support_email` stuff from the dev section to the "Testing" section (where it's needed first) and then referred back to it in the dev section.

Gave a full command example for testing scenario --- I'll be needing it this week! :-)

Clarified need to create the folder for the ACME verification files in some Node.js/Python scenarios, based on your explanations.

Added link for YAML format.

will-in-wi commented:
Thanks!
